CHANGELOG.md

```diff
@@ -4,7 +4,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [2.0.1] - 2017-02-26
+## [2.0.2] - 2018-03-09
+### Fixed
+- \#3: Corrected an issue that caused the plugin not to load items on initialization if the `filter` option was set to `true` 
```

Comment:
Remove '\\'

Reply:
The '\' is required otherwise the '#' is considered a heading by MD.

Reply:
Setting `searchQuery` to `null` resolves the issue because `null` is different from an empty string which is passed by the filter input and compared to `searchQuery`.